src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -369,6 +373,36 @@ lemma sum_apply {ι : Type*} (s : finset ι) (f : ι → normed_group_hom V₁ V
   (∑ i in s, f i) v = ∑ i in s, (f i v) :=
 by simp only [coe_sum, finset.sum_apply]
 
+lemma sum.norm_le {ι : Type*} (s : finset ι)
```

Comment:
```suggestion
lemma norm_sum_le {ι : Type*} (s : finset ι)
```
When using the operator norm, I think this should just be a combination of `norm_sum_le` for normed groups combined with `finset.sum_le_sum`. So maybe it's not even really worth stating it separately.
I think this is just a remnant of my stupid `bound_by` definition.

Reply:
Even better: we already have `norm_sum_le_of_le`, so this is completely useless (I am removing it also from LTE).